build.txt

```diff
@@ -1,6 +1,6 @@
 ﻿displayName = Which Mod Is This From? (WMITF)
 author = gardenapple
-version = 2.6.3
+version = 3.0.0
```

Comment:
Again, not sure why the major version change.

Reply:
Just wanted it to be distinct. I'll change the second number if that is bothersome. I am aware that it is SemVer.